orbit/cmd/desktop/desktop.go

```diff
@@ -247,6 +249,11 @@ func main() {
 		}()
 
 		if runtime.GOOS == "darwin" {
+			homedir, err := os.UserHomeDir()
+			if err != nil {
+				log.Fatal().Err(err).Msg("failed to get user's home directory")
+			}
+			mrw := migration.NewReadWriter(path.Join(homedir, "Library", constant.MigrationFileName))
```

Comment:
this is cool, but I was thinking about looking for a suitable folder inside `~/Library`, for example `~/Library/Caches` or `~/Library/Preferences`

ideally move this into a function in `orbit/pkg` or similar so it's our go-to for preferences or cache

Reply:
Moved to `~/Library/Caches/com.fleetdm.orbit`!